src/components/__tests__/TopNav.test.tsx

```diff
@@ -1,65 +1,59 @@
 import React from "react";
 import { shallow } from "enzyme";
 import TopNav from "../TopNav";
+import Button from "@mui/material/Button";
 
 // TODOs: need to figure out the wrapper type before using the following beforeEach code block
 // let wrapper;
 // beforeEach(()=>{
 // 	wrapper = shallow(<TopNav/>);
 // })
 
+const links = [
+    {
+        to: "/posts/general",
+        text: "General",
+    },
+    {
+        to: "/posts/learn",
+        text: "Learn",
+    },
+    {
+        to: "/posts/interview",
+        text: "Interview",
+    },
+    {
+        to: "/posts/project",
+        text: "Project",
+    },     
+    {
+        to: "/login",
+        text: "SignIn",
+    },
+    {
+        to: "/register",
+        text: "SignUp",
+    },
+    {
+        to: "/my-account/1",
+        text: "Profile",
+    },
+];
+
 // test component rendering
-describe("<TopNav /> rendering", () => {
+describe("<TopNav />", () => {
     const wrapper = shallow(<TopNav />);
     // test if the TopNav component renders
     it("should be able to render <TopNav />", () => {
         wrapper;
```

Comment:
Does this actually test that the component renders? Expect and matchers are used for testing

Reply:
That part was actually Lisa's test.